lib/net/imap.rb

```diff
@@ -786,6 +799,15 @@ def expunge
       end
     end
 
+    # Similar to #expunge, but takes a set of unique identifiers as
+    # argument.
+    def uid_expunge(set)
+      synchronize do
+        send_command("UID EXPUNGE", MessageSet.new(set))
```

Comment:
No multi-line suggestions, yet! 😉
```suggestion
        send_command("UID EXPUNGE", MessageSet.new(uid_set))
```